sdk/program/Cargo.toml

```
@@ -32,8 +32,8 @@ serde_bytes = { workspace = true }
 serde_derive = { workspace = true }
 sha2 = { workspace = true }
 sha3 = { workspace = true }
-solana-frozen-abi = { workspace = true }
-solana-frozen-abi-macro = { workspace = true }
+solana-frozen-abi = { workspace = true, optional = true }
+solana-frozen-abi-macro = { workspace = true, optional = true}
```

```suggestion
solana-frozen-abi-macro = { workspace = true, optional = true }
```
I think `cargo sort` will complain about this